### PR TITLE
Fix typos and sponsor heading font size and apply now button

### PR DESCRIPTION
**Pull request by gracelu0**
- Fix typos on About and FAQ pages and make sponsor title font size consistent at 48px
- Removed Jeanette and added Hussain to execs
- Change 'Apply now' button to 'Applications open on Dec 27th':
Screen Shot 2021-12-23 at 11 58 15 PM

### CODE REVIEW

**Review by praneershrest — COMMENTED** [review]
Instead of deleting the changes, just comment it out (for applications especially) so we can revert back when applications open :)

File: src/pages/About/index.tsx [hunk 1]
```diff
@@ -44,17 +44,17 @@ const About = () => {
...
           <p className={styles.paragraph}>
             StormHacks 2021 brought together 300+ hackers from 146 universities,
             with 45% of participants being first-time hackers. StormHacks is
             SFU's largest hackathon and an MLH Member Event for the 2022 Season.
           </p>
           <a className={styles.applyBtn} href="/">
-            Apply Now
-            <EditIcon className={styles.editIcon} />
+            Applications open Dec 27th
+            {/* TODO: uncomment when applications are open <EditIcon className={styles.editIcon} /> */}
```
Comment (praneershrest) [inline src/pages/About/index.tsx]: 'Apply Now' missing in comment

Comment (gracelu0) [reply src/pages/About/index.tsx]: oops, updated

**Review by praneershrest — APPROVED** [review]
lgtm

**Pull request merged**